src/slack.coffee

```diff
@@ -73,7 +73,17 @@ class SlackBot extends Adapter
       if id is user.name then delete @robot.brain.data.users[user.id]
 
   userChange: (user) =>
-    newUser = {name: user.name, real_name: user.real_name, email_address: user.profile.email}
+    newUser =
+      name: user.name
+      real_name: user.real_name
+      email_address: user.profile.email
+      slack: {}
+    for key, value of user
+      # user contains an of the SlackClient, which and contains references to the all the data types (users, channels) plus things like the token, s
```

Comment:
I edited this a few times, and ended up with word soup :sweat: Here's what I think I was trying to say originally:

> user contains an *instance* of the Slack, which ~~contains~~ references to all the data type.

I'll take another pass at describing it.